src/main/java/application/controllers/ZoomFactory.java

```diff
@@ -0,0 +1,30 @@
+package application.controllers;
+
+import javafx.scene.Group;
+import javafx.scene.control.ScrollPane;
+
+/**
+ * Created by Daphne van Tetering on 3-5-2016.
+ */
+public class ZoomFactory {
+
+    /**
+     * zoom factory oonstructor method.
+     */
+    public ZoomFactory() {
```

Comment:
Constructor should be private, as ZoomFactory should probably be a static class.